WellnessLiving/Wl/Location/View/ViewModel.php

```diff
@@ -309,6 +309,13 @@ class ViewModel extends WlModelAbstract
    */
   public $url_site = '';
 
+    /**
+     * Twitter url.
+     *
+     * @var string

```

Comment:
Is this returned as part of the get? It should have the '@get result' tag then.

Reply:
Fixed